mlflow/llama_index/pyfunc_wrapper.py

```diff
@@ -236,7 +236,7 @@ def _run():
                 finally:
                     loop.close()
 
-            thread = threading.Thread(target=_run)
+            thread = threading.Thread(target=_run, name="llamaindex_async_task_runner")
```

Comment:
In `threading.Thread` in the mlflow directory, can we use `uuid` to make sure the name is unique? Here's an example:


```python
thread = threading.Thread(target=my_function, name=f"worker_{uuid.uuid4().hex[:8]}")
```

Reply:
looks like the commit wasn't pushed successfully